pages/api/slide/delete.ts

```diff
@@ -0,0 +1,30 @@
+import { Types } from "mongoose";
+import { NextApiRequest, NextApiResponse } from "next";
+import { ErrorResponse } from "types/ErrorResponse";
+import { Module } from "../models/module";
+import { Slide } from "../models/slide";
+
+const del = async (
+    req: NextApiRequest,
+    res: NextApiResponse<ErrorResponse>,
+): Promise<void> => {
+    const { id, sid } = req.query;
```

Comment:
Do we need both slide ID and module ID here? Maybe we add a Module ID inside each slide instead?